index.html

```diff
@@ -0,0 +1,205 @@
+      </section>
+
+      <!-- works and portfolio section! -->
+      <section class="boxworks">
+        <!-- 1st work! -->
+        <div class="works">
+          <img src="images/works/work1.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul>
+            <li class="worksubtitle">CANOPY</li>
+            <li class="bullets">Back End Dev</li>
+            <li class="bullets">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+
+        <!-- 2nd work! -->
+        <div class="works">
+          <img src="images/works/work2.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul>
+            <li class="worksubtitle">CANOPY</li>
+            <li class="bullets">Back End Dev</li>
+            <li class="bullets">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+
+        <!-- 3rd work! -->
+        <div class="works">
+          <img src="images/works/work3.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul>
+            <li class="worksubtitle">CANOPY</li>
+            <li class="bullets">Back End Dev</li>
+            <li class="bullets">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+
+        <!-- 4rt work! -->
+        <div class="works">
+          <img src="images/works/work4.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul>
+            <li class="worksubtitle">CANOPY</li>
+            <li class="bullets">Back End Dev</li>
+            <li class="bullets">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+      </section>
+
+      <!-- About me section! -->
+      <section class="aboutme">
+        <div class="title">About Myself</div>
```

Comment:
- [ ] Kindly change this `div` into an `h1` tag, as this text it's actually a title, let's try to keep it as semantic as possible!

Reply:
Changes done!